### ISSUE TITLE
Couldn't import Forminho components even if it's installed by different ways

### ISSUE THREAD

**Issue by caiookb** [CarlosRafael22/forminho#21]
Hello, Carlos

I had some problem by installing the dependency using `link` from _yarn_ or _npm_ so I found another way to do it, making you dependency as a pack to be installed by a package manager, but I also had some problems by doing it. 

So, I was trying to follow the readme steps to install the dependency but at the second step, I couldn't make progress, even with the links (I tried by npm and yarn, I also found that those "links" has a bunch of issues) the package manager couldn't find  "_forminho_" and would never import the dependency on my project. But after a while of trying different ways of using links, I finally could install it, but I had some problems with importing Forminho component, so I tried to install it with a pack.

So I tried those steps to install with a pack:

1.  Clone the repository and go to the most recent branch;
2.  Run `npm install` to install all "forminho" dependencies;
3.  Run `npm build `;
4.  Run `npm pack` (It creates a .tgz so you can import on your package.json and install normally);
5.  Copy the .tgz to a temp folder on your project root folder;
6.  Add `"forminho": "file:/./tmp/forminho-0.0.1.tgz"`, for example, in your package.json;
7.  Run `npm install` on your project folder;

Then I tried to import forminho on my project 
```javascript
import { Forminho, Form, Field, Button } from "forminho"
```
Form, Field and Button are working properly, Forminho is imported as well but with problems, it says that the module '"forminho"' has no exported member 'Forminho',

So I tried to not use the Forminho component, using only Form and the others, it worked fine, something like this:

![Form](https://user-images.githubusercontent.com/46979876/111730069-597b9f80-884f-11eb-87a7-8bc42f5458c2.png)

```javascript
   <Form
      initialValues={{email: ""}}
      onSubmitHandler={onSubmitHandler}
      onValidationHandler={onValidationHandler}
      submitButtonText="Confirm signup"
    >
      <Field.Input type="email" name="email" placeholder="Enter email" />
  </Form>
```

I had the same problem by using both ways to install.

What problems I'm going to have If I don't use the Forminho tag?

Screenshot:

![Forminho](https://user-images.githubusercontent.com/46979876/111729758-95facb80-884e-11eb-9afd-1f052a3d516a.png)

**Comment by CarlosRafael22** [comment]
Hi Caio, thank you so much for the issue!

I really appreciate your effort to install forminho lib and make it work on your machine. I didn't know about this strategy to install the package so thanks for that! Also, I will put it as the recommended way to install the lib in the local machine.

Regarding the Forminho component problem, yesterday I removed it to simplify the use of the library but haven't updated the documentation yet. Thanks for pointing that out! I will update the documentation today to "fix" this problem.

I opted to remove `Forminho` component because after removing the `LiveValue` component it wouldn't be necessary to have it anymore. The `Forminho` component was the context provider for all the children components. Since the `LiveValue` component could have access to a input value outside the `Form` through the access of the context, I need to have the `Forminho` to give the context to the `LiveValue`.

After refactoring, removing the `LiveValue` component and creating the `liveUpdate` prop in the `Field`, I no longer needed the `Forminho` to have the context and passed it to the `Form`. This way the Forminho component was unecessary and I removed it.

Thanks for reporting! I will close the issue after I update the documentation :)
